packages/lesswrong/components/sunshineDashboard/SunshineCuratedSuggestionsList.tsx

```diff
@@ -3,25 +3,40 @@ import { Components, registerComponent } from '../../lib/vulcan-lib';
 import { useMulti } from '../../lib/crud/withMulti';
 import { Posts } from '../../lib/collections/posts';
 
-const SunshineCuratedSuggestionsList = ({ terms }) => {
+const SunshineCuratedSuggestionsList = ({ terms, belowFold }:{
+  terms: any,
+  belowFold?: boolean
+}) => {
   const { results, loading, count, totalCount, loadMore, showLoadMore } = useMulti({
     terms,
     collection: Posts,
     fragmentName: 'PostsList',
     enableTotal: true,
     itemsPerPage: 60
   });
-  
+
+  const { results: curatedResults } = useMulti({
+    terms: {view:'curated', limit:1},
+    collection: Posts,
+    fragmentName: 'PostsList',
+  });
+  const curatedDate = new Date(curatedResults && curatedResults[0]?.curatedDate)
+  const twoDaysAgo = new Date(new Date().getTime()-(2*24*60*60*1000));
```

Comment:
Causes an SSR mismatch if the time between when SSR completes and when the client rerenders straddles the 2-day boundary. Fixing that would involve adding significant extra infrastructure (injecting a `now` timestamp into the SSRed document, and passing it around with a context), so it's not worth fixing here, but this is something we'll want to address eventually.

Reply:
Gotcha. This only applies to admins tho, right? If so that seems basically fine, at least for the  foreseeable future.